### ISSUE TITLE
Incomplete typing for AsyncConnection.execute's parameters

### ISSUE THREAD

**Issue by squaresmile** [sqlalchemy/sqlalchemy2-stubs#165]
**Describe the bug**
`AsyncConnection.execute`'s `parameters` typing info is missing `Sequence[Mapping[Any, Any]]`

**Expected behavior**
The following example to not have type error.

**To Reproduce**
```py
import asyncio
from sqlalchemy import Table, MetaData
from sqlalchemy.ext.asyncio import create_async_engine

meta = MetaData()
t1 = Table("t1", meta)

async def async_main() -> None:
    engine = create_async_engine(
        "postgresql+asyncpg://scott:tiger@localhost/test",
        echo=True,
    )

    async with engine.begin() as conn:
        await conn.run_sync(meta.drop_all)
        await conn.run_sync(meta.create_all)

        await conn.execute(
            t1.insert(), [{"name": "some name 1"}, {"name": "some name 2"}]
        )
    await engine.dispose()

asyncio.run(async_main())
```

**Error**

```
> mypy --strict .\test.py
test.py:20: error: Argument 2 to "execute" of "AsyncConnection" has incompatible type "List[Dict[str, str]]"; expected "Optional[Mapping[Any, Any]]"
Found 1 error in 1 file (checked 1 source file)
```

**Versions.**
 - OS: Windows 10
 - Python: 3.9.6
 - SQLAlchemy: 1.4.23
 - mypy: 0.910
 - SQLAlchemy2-stubs: 0.0.2a13

**Have a nice day!**

**Comment by CaselIT** [comment]
Thanks for reporting

**Comment by CaselIT** [comment]
Released as v0.0.2a15
